hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/timeline/versioning/v1/TimelineArchiverV1.java

```diff
@@ -400,7 +400,8 @@ private boolean deleteArchivedInstants(List<HoodieInstant> archivedInstants, Hoo
       completedInstants.stream()
           .forEach(instant -> activeTimeline.deleteInstantFileIfExists(instant));
     }
-
+    // Call Table Format archive to allow archiving in supplementary table format.
+    table.getMetaClient().getTableFormat().archive(archivedInstants, table.getContext(), table.getMetaClient(), table.getViewManager());
```

Comment:
same. not a fan of these extra calls everywhere, to just call the "supplementary" table format